### PR TITLE
[#20261] Add support 1C:Enterpise syntax highlight

### PR DESCRIPTION
**Pull request by Kirill**
Add support for extension .bsl, .os (OneScript oscript.io)

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/20261

Sorry, I'm not sure that correct. I take PR https://github.com/mattermost/mattermost-webapp/pull/5468 and did changes like this PR.

#### Release Note
```release-note
Added syntax highlighting for 1C:Enterprise (BSL)
```

### CODE REVIEW

**Review by BenCookie95 — APPROVED** [review]

**Comment by Kirill** [comment]
I look this test server, and my pr not working. Syntax not colorize
ss_20220527_065302_8
I try check script working in simple html at evening. I don't know how check and why don't working in Mattermost.

**Comment by Kirill** [comment]
Filename begin for number. I think my first commit not work when I rename 1c to bsl. When I change bsl to 1c, lint not working. How I can fix it? Sorry, I haven't knowledge about JS and TS.

File: utils/constants.tsx [hunk 1]
```diff
@@ -1612,6 +1612,7 @@ export const Constants = {
     } as Record<string, [string, number]>),
     CODE_PREVIEW_MAX_FILE_SIZE: 500000, // 500 KB
     HighlightedLanguages: {
+        1c: {name: '1C:Enterprise', extensions: ['bsl', 'os'], aliases: ['bsl']},
```
Comment (hmhealey) [inline utils/constants.tsx:1615]: You just have to wrap the field name in quotation marks if it starts with a number or contains anything that can't normally be the name of a variable.

```suggestion
        '1c': {name: '1C:Enterprise', extensions: ['bsl', 'os'], aliases: ['bsl']},
```

**Comment by hmhealey** [comment]
With that change, it works for me on my test server too

Screen Shot 2022-06-08 at 11 31 28 AM

**Comment by Kirill** [comment]
@hmhealey, thank you very much. I push new commit to repo. wait result.

**Comment by Kirill** [comment]
It's work for me... 
image

**Review by hmhealey — APPROVED** [review]
Works for me too now

**Comment by jgilliam17** [comment]
@hmhealey Can you please review and approve the workflow below so I can trigger e2es? Thanks 🙂

**Comment by jgilliam17** [comment]
Thanks @Kirill 
Tested, looks good to merge - pending E2Es.
- Verified syntax highlighting.

**Comment by hmhealey** [comment]
Sorry about that. I always forget that that one needs to be ran manually

**Review by jgilliam17 — APPROVED** [review]
[E2E report](https://mattermost-cypress-report.s3.amazonaws.com/209079-94aa632-onprem-ent-webapp-pr-10360-mm-ee-test:94aa632/mochawesome.html) doesn't have any PR related failures. LGTM

**Comment by jgilliam17** [comment]
Jira ticket added for QA release testing: https://mattermost.atlassian.net/browse/MM-45210

**Pull request merged**